<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page400</class>
 <widget class="QWidget" name="page400">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">page400</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#page400
{
}</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item 1:</string>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButton_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string notr="true"/>
   </property>
   <property name="statusvar">
    <string notr="true"/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pressText">
    <string/>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string notr="true"/>
   </property>
   <property name="statusvar">
    <string notr="true"/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pressText">
    <string/>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item 2:</string>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButton_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string notr="true"/>
   </property>
   <property name="statusvar">
    <string notr="true"/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pressText">
    <string/>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item 3:</string>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButton_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string notr="true"/>
   </property>
   <property name="statusvar">
    <string notr="true"/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pressText">
    <string/>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item 4:</string>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButton_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>330</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string notr="true"/>
   </property>
   <property name="statusvar">
    <string notr="true"/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pressText">
    <string/>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item 5:</string>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButtonBack">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string notr="true">HOME</string>
   </property>
   <property name="statusvar">
    <string notr="true"/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pressText">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="systemicons.qrc">
     <normaloff>:/systemicons/Go_back.png</normaloff>:/systemicons/Go_back.png</iconset>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButton_SaveList">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string notr="true"/>
   </property>
   <property name="statusvar">
    <string notr="true"/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pressText">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="systemicons.qrc">
     <normaloff>:/systemicons/Save.png</normaloff>:/systemicons/Save.png</iconset>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ATCMbutton</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="systemicons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>atcmButtonBack</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page400</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goto_page(const char*,bool)</slot>
 </slots>
</ui>
